core/put/source/dev_ui.h

```diff
@@ -63,9 +63,11 @@ namespace put
                 texture_cube_array = pen::TEXTURE_COLLECTION_CUBE_ARRAY
             };
         }
-        typedef e_ui_shader::ui_shader_t ui_shader;
 
-        bool        init();
+        typedef e_ui_shader::ui_shader_t ui_shader;
+        ImGuiStyle  &default_pmtech_style();
```

Comment:
the rest of the code style would go like this:
```c++
ImGuiStyle&   default_pmtech_style();
```

With the reference or pointers touching the typename.